sql/core/src/test/scala/org/apache/spark/sql/execution/adaptive/AdaptiveQueryExecSuite.scala

```diff
@@ -780,5 +780,14 @@ class AdaptiveQueryExecSuite
       }
     }
   }
+
+  test("SPARK-31242: clone SparkSession should respect sessionInitWithConfigDefaults") {
```

Comment:
I know it's not directly related to AQE. But the issue can not be reproduced without `SharedSparkSessionBase` and seems AQE now is more likely related to `cloneSession`. So I decide to add it here.

Reply:
how about `SQLQuerySuite`?

Reply:
sounds good.